FSEParser_V3.1.py

```diff
@@ -74,216 +72,229 @@
+    """Get needed options for processing"""
+    usage = "usage: %prog -c CASENAME -q REPORT_QUERIES -s SOURCEDIR -o OUTDIR"
+    options = OptionParser(usage=usage)
 
     options.add_option("-c",
-                   action="store",
-                   type="string",
-                   dest="casename",
-                   default=True, 
-                   help="The name of the current session, used for naming standards");
+                       action="store",
+                       type="string",
+                       dest="casename",
+                       default=True,
+                       help="The name of the current session, used for naming standards")
     options.add_option("-q",
-                   action="store",
-                   type="string",
-                   dest="report_queries",
-                   default=True, 
-                   help="The location of the report_queries.json file containing custom report queries to generate targeted reports");
+                       action="store",
+                       type="string",
+                       dest="report_queries",
+                       default=True,
+                       help="The location of the report_queries.json file containing custom report "
+                            "queries to generate targeted reports")
     options.add_option("-s",
-                   action="store",
-                   type="string",
-                   dest="sourcedir",
-                   default=True, 
-                   help="The source directory containing fsevent files to be parsed");
+                       action="store",
+                       type="string",
+                       dest="sourcedir",
+                       default=True,
+                       help="The source directory containing fsevent files to be parsed")
     options.add_option("-o",
-                   action="store",
-                   type="string",
-                   dest="outdir",
-                   default=True, 
-                   help="The destination directory used to store parsed reports");
+                       action="store",
+                       type="string",
+                       dest="outdir",
+                       default=True,
+                       help="The destination directory used to store parsed reports")
     # Return options to caller #
-    return options;
-
-def ParseOptions():
-        # Get options
-        options = GetOptions()
-        (opts,args) = options.parse_args()
-
-        # The meta will store all information about the arguments passed #
-        meta = {
-            'casename':opts.casename,
-            'reportqueries':opts.report_queries,
-            'sourcedir':opts.sourcedir,
-            'outdir': opts.outdir
-            }
-
-        # Test arguments passed #
-        if opts.report_queries==True:
-            options.error("Unable to proceed. report_queries.json was not specified using the -q option. \
+    return options

```

Comment:
# Test if Python version in use is 3 or greater
if (sys.version_info > (3, 0)):
    print 'Error: Unable to run script. FSEventsParser does not currently support Python 3.x syntax.'
    print 'Python 2.7 is recommended.'
    sys.exit(0)